autorally_core/launch/stateEstimator.launch

```diff
@@ -1,19 +1,48 @@
+<?xml version="1.0"?>
+
+
+<!-- singlePlatform.launch
+
+Use Gazebo to simulate a vehicle with Ackermann steering.
+
+Arguments:
+    sim (boolean, default: false)
+        parameter used to run state estimator in simulation.
+    InvertZ (boolean, default: true)
+        if the state estimator should invert the Z axis of the imu
+    InvertY (boolean, default: true):
+        if the state estimator should in the Y axis of the imu
+    namespace (string, default: "")
+        the namespace of the topic published by gazebo
+    FixedInitialPose (boolean, default: false)
+        if a fixed initial pose should be used
+-->
+
 <launch>
 
+
+
   <include file="$(find autorally_core)/launch/hardware.machine" />
   <include file="$(find autorally_core)/launch/wheel_odometry.launch" />
+  <arg name="sim" default="false"/>
+  <arg name="namespace" default=""/>
+  <arg name="InvertY" default="true"/>
+  <arg name="InvertZ" default="true"/>
+  <arg name="FixedInitialPose" default="false"/>
+
+  <param name="/gps_imu/FixedInitialPose" value="$(arg sim)"/>
 
 
   <node name="state_estimator" pkg="autorally_core" type="StateEstimator" output="screen" machine="autorally-master">
-      <remap from="/state_estimator/gps" to="/gpsRoverStatus"/>
-      <remap from="/state_estimator/imu" to="/imu/imu"/>
-      <remap from="/state_estimator/pose" to="/pose_estimate"/>
-      <remap from="/state_estimator/filter" to="/imu/filter"/>
-      <remap from="/state_estimator/bias_acc" to="/bias_acc"/>
-      <remap from="/state_estimator/bias_gyro" to="/bias_gyro"/>
-      <remap from="/state_estimator/wheel_odom" to="/wheel_odom"/>
-      <remap from="/state_estimator/status" to="/pose_estimate/status"/>
-      
+      <remap from="$(arg namespace)/state_estimator/gps" to="$(arg namespace)/gpsRoverStatus"/>
```

Comment:
The addition of this namespace means that on the robot, everyone will now have to subscribe to a dafferent topic depending on the platform?

Could the default namespace here be empty so that on the real robots we can maintain the current topic names?

Reply:
I can actually just remove namespace as a parameter